orc8r/gateway/python/scripts/register.py

```diff
@@ -77,7 +102,7 @@ def main():
     parser.add_argument(
         "--ca-file",
         type=str,
-        help="orc8r's root CA file",
+        help="orc8r's root CA file contents",
```

Comment:
I think let's leave this the way it was before actually -- it's kinda awkward to paste the entire root CA contents into the CLI cmd. (Or did you have thoughts on why this is preferable?)

Reply:
I was just thinking that we never really use the contents from the response body `registration_info: {"rootCA": "...", ....}` and it should probably get passed in somewhere. 
Before I assumed it was taking in a filepath, but does the `rootCA.pem` live anywhere but the `/var/opt/magma/certs` directory? I think it might be weird for the operator to know to paste the `rootCA.pem` to a random location and then point to this file through the script? Idk lemme know your thoughts.

Reply:
Yeah so the idea is they would save the root CA cert to the file system, then point to it from the cli invocation. I think that's a bit cleaner, just cause pasting the whole contents of a cert into a cli invocation is a bit much

(The "win" is, if we can get moved over to using cert-manager, AWS-deployed Orc8rs can skip this step)